content/en/logs/log_configuration/attributes_naming_convention.md

```diff
@@ -110,15 +110,26 @@ Typical integrations relying on these attributes include [Apache][4], [Varnish][
 
 The following attributes are related to the geolocation of IP addresses used in network communication. All fields are prefixed by `network.client.geoip` or `network.destination.geoip`.
 
-| **Fullname**                                | **Type** | **Description**                                                                                                                      |
-| :------------------------------------------ | :------- | :----------------------------------------------------------------------------------------------------------------------------------- |
-| `network.client.geoip.country.name`         | `string` | Name of the country                                                                                                                  |
-| `network.client.geoip.country.iso_code`     | `string` | [ISO Code][6] of the country (example: `US` for the United States, `FR` for France)                                                  |
-| `network.client.geoip.continent.code`       | `string` | ISO code of the continent (`EU`, `AS`, `NA`, `AF`, `AN`, `SA`, `OC`)                                                                 |
-| `network.client.geoip.continent.name`       | `string` | Name of the continent (`Europe`, `Australia`, `North America`, `Africa`, `Antartica`, `South America`, `Oceania`)                    |
-| `network.client.geoip.subdivision.name`     | `string` | Name of the first subdivision level of the country (example: `California` in the United States or the `Sarthe` department in France) |
-| `network.client.geoip.subdivision.iso_code` | `string` | [ISO Code][6] of the first subdivision level of the country (example: `CA` in the United States or the `SA` department in France)    |
-| `network.client.geoip.city.name`            | `String` | The name of the city (example `Paris`, `New York`)                                                                                   |
+`network.client.geoip.country.name` 
+: Type: `string` <br> Name of the country. 
+
+`network.client.geoip.country.iso_code` 
+: Type: `string` <br> [ISO Code][6] of the country (example: `US` for the United States, `FR` for France).
```

Comment:
Is this link intended to open the unified service tagging doc? I found the same line in [this doc](https://docs.datadoghq.com/real_user_monitoring/browser/data_collected/#geo-location), which opens to the Wikipedia page about ISO:
https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes

Reply:
Thanks, fixed!